Sources/SwiftParser/Statements.swift

```diff
@@ -943,14 +943,13 @@ extension Parser.Lookahead {
       return false
     }
 
-    let next = peek()
-
     // If 'then' is followed by a binary or postfix operator, prefer to parse as
     // an expr.
-    if BinaryOperatorLike(lexeme: next) != nil || PostfixOperatorLike(lexeme: next) != nil {
+    if nextTokenAt(anyIn: BinaryOperatorLike.self) != nil || nextTokenAt(anyIn: PostfixOperatorLike.self) != nil {
       return false
     }
 
+    let next = peek()
```

Comment:
If you’re already touching this, you can just remove `next` completely and replace both usages by `peek()`. Calling `peek` is cheap enough that we don’t need to store it in a variable.